### PR TITLE
Feature/pass moveit capabilities by arg

### PR DESCRIPTION
**Pull request by martiniil**
Move Group allows the definition of capabilities that are loaded. With this PR these capabilities can be defined via a launch-file argument.

### CODE REVIEW

**Comment by jschleicher** [comment]
What about documentation?

Would you like to add the new option to our main [Readme.md](Readme.md)? Or create a readme for prbt_moveit_config which links to the upstream training tutorial (e.g. [ROS-I training RViz](http://ros-industrial.github.io/industrial_training/_source/session3/Motion-Planning-RVIZ.html) and adds the additional new parameter)?

**Review by jschleicher — CHANGES_REQUESTED** [review]
Is this parameter useful for the end-user? Then it would need some documentation. Or is it for tests only?

**Comment by agutenkunst** [comment]
I would say it is useful if you know what your doing and provide this argument for more advanced users.

Currently there are multiple move_group settings:
```
  <!-- move_group settings -->
  <arg name="allow_trajectory_execution" default="true"/>
  <arg name="fake_execution" default="false"/>
  <arg name="max_safe_path_cost" default="1"/>
  <arg name="jiggle_fraction" default="0.05" />
  <arg name="publish_monitored_planning_scene" default="true"/>
  <arg name="moveit_manage_controllers" default="true"/>
```

Since currently none of these move_group settings is documented in our Readme I would be in favour of keeping this PR as it is.

**Comment by agutenkunst** [comment]
I just opened https://github.com/ros-planning/moveit/pull/998

Maybe this will go upstream in moveit. Waiting for approval / further discussions.

**Review by jschleicher — APPROVED** [review]
approved. Waiting for travis...

**Pull request merged**
